doc/source/tune/faq.rst

```diff
@@ -733,3 +733,59 @@ If `grid_search` is provided as an argument, the grid will be repeated ``num_sam
 Note that search spaces may not be interoperable across different search algorithms.
 For example, for many search algorithms, you will not be able to use a ``grid_search`` or ``sample_from`` parameters.
 Read about this in the :ref:`Search Space API <tune-search-space>` page.
+
+.. _tune-working-dir:
+
+How do I access relative filepaths in my Tune training function?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's say you launch a Tune experiment from ``~/code/my_script.py``. By default, Tune
+changes the working directory of each worker from ``~/code`` to its corresponding trial
+directory (e.g. ``~/ray_results/exp_name/trial_0000x``). This guarantees separate working
+directories for each worker process.
+
+There are two main reasons for this default:
+
+1. If trial writes artifacts, we want to avoid conflicts: if two trials on the same machine write to ``artifact.bin``, the second one would overwrite the artifact from the first.
+
+2. In distributed settings with more than one machine, we cannot guarantee that the working directory from the driver script exists on the other remote nodes. However, we can guarantee that the trial directory exists.
```

Comment:
How about removing these two points? I don't think the level of details is useful for users. The statement that we guarantee separate working directories [to ensure output files don't conflict] is sufficient.